OpenEdXMobile/src/main/java/org/edx/mobile/view/MainDashboardActivity.java

```diff
@@ -55,7 +56,8 @@ public static Intent newIntent() {
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-
+        NotificationUtil.subscribeToTopics(environment, this);
+        NotificationUtil.logFirebaseToken();
```

Comment:
Is this the right place for this code? Shouldn't this be in the `MainApplication` class and guarded by the `Config#areFirebasePushNotificationsEnabled` check?